### DISCUSSION TITLE
Welcome to BookPlayer Discussions!

### DISCUSSION THREAD

**Discussion by pichfl** [TortugaPower/BookPlayer#524] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by crozo5** [comment]
Is it in your roadmap to be able to synchronize the audiobooks on the apple watch and be able to play them directly from the apple watch without having to have the iphone nearby?

Thanks

**Reply by GianniCarlo** [reply]
Hey @crozo5 ! we do plan to have it, but it's way down the pipeline to be honest, our current main priorities are:
* Folder support
* Player redesign
* Cross-device synchronization (via iCloud maybe)

These three contain a lot of things to do, and we'll be splitting them up (when possible) in multiple releases, until we achieve those goals. We'll also work on smaller things alongside the main goals, but getting back to the point, synchronizing books between the Watch and the main App will be easier once we achieve the cross-device synchronization milestone, and it's relegated to being worked on after that

**Comment by Dahveed7** [comment]
**Getting Files Into BookPlayer**

I'm trying to use iTunes file sharing to get my files into BookPlayer. I could have sworn that is how I did it before but it doesn't seem to work now. The audiobook folder SSNL gets copied somewhere but it doesn't show up in BookPlayer. Am I missing something??

![Screen Shot 2021-04-08 at 9 43 02 PM](https://user-images.githubusercontent.com/82148605/114005575-c5cc4a80-98b3-11eb-89f7-ca4ad3d9358f.jpg)

**Reply by GianniCarlo** [reply]
Yeah that's something we've had to disable due to a bug, current workaround is to zip the folder, and transfer that zip file 👍
